examples/lambda-layer/invoke.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+echo "Enter your Lambda's name (usually in the form mylambda-f0780fd95f4) followed by [ENTER]: "
+read LAMBDA
+echo "Attempting to invoke $LAMBDA ..."
+aws lambda invoke --function-name $LAMBDA --log-type Tail out.txt | jq '.LogResult' -r | base64 --decode
```

Comment:
I don't think we use this file anymore - it was a very old artifact of back when these really were "examples".  And for the Layer in particular, this won't even work.

Reply:
gone - good spot!